<!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Engineering | Vsycle</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link rel="icon" href="../favicon.ico"><link href="../css/content.css" rel="stylesheet"></head><body><!-- # TODO: #6 Fix table caption font--><!-- # TODO: #7 Fix citations links font size--><nav class="navbar navbar-expand-xl fixed-top-fix"><div class="container d-flex justify-content-between"><a class="navbar-brand d-lg-inline-block" href="https://vsycle.github.io/wiki/"><img src="../assets/img/logo.png"></a><button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button><div class="collapse navbar-collapse" id="navbarNav"><ul class="navbar-nav ml-auto"><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarProjectDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Project</a><div class="dropdown-menu" aria-labelledby="navbarProjectDropdown"><a class="dropdown-item" href="https://vsycle.github.io/wiki/Description">Description</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Clouds">Clouds</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Engineering">Engineering</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Dispersion">Dispersion</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Legislation">Legislation</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Contribution">Contribution</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/SOPs">SOPs</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Safety">Safety</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Notebook">Notebook</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarHuman Practices &amp; OutreachDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Human Practices &amp; Outreach</a><div class="dropdown-menu" aria-labelledby="navbarHuman Practices &amp; OutreachDropdown"><a class="dropdown-item" href="https://vsycle.github.io/wiki/Human_Practices">Human Practices</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Outreach">Outreach</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Implementation">Implementation</a></div></li><li class="nav-item dropdown"><a class="nav-link dropdown-toggle" id="navbarTeamDropdown" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Team</a><div class="dropdown-menu" aria-labelledby="navbarTeamDropdown"><a class="dropdown-item" href="https://vsycle.github.io/wiki/Team_Members">Team Members</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Collaborations">Collaborations</a><a class="dropdown-item" href="https://vsycle.github.io/wiki/Attributions">Attributions</a></div></li><li class="nav-item"><a class="nav-link" href="https://vsycle.github.io/wiki/Awards">Awards</a></li><li class="nav-item"><a class="nav-link" href="https://vsycle.github.io/wiki/Fun">Fun</a></li></ul></div></div></nav><header class="d-flex justify-content-center align-items-center"><div class="banner banner-project"><div class="container"><h1>Engineering</h1><p class="lead pl-1">Project</p></div></div></header><main><div class="container"><div class="row"><div class="sidebar col-lg-3"><div class="nav" id="contents"><h5>Contents</h5><ul></ul></div></div><div class="content col-lg-9"><article><h1>Introduction</h1><p>In module engineering, we investigated the optimal method for producing a biological cloud seeding agent containing <a href="https://2020.igem.org/Team:UGent_Belgium/Clouds#Icenucleationproteins(INPs)">ice nucleation protein (INP)</a>. First, we selected the most promising INP candidate. Next, an easy and efficient expression system to functionally express INPs on the bacterial membrane was optimized. As chassis organism, we use <em>E. Coli</em> cells. The INP gene of <em>Pseudomonas syringae</em>, InaZ <a href="#citation5">Han et al., 2017</a>, is the most investigated INP and will be expressed in these <em>E. coli</em> cells. Besides, this INP is already successfully expressed both attached to the <em>E. coli</em> outer cell membrane as well as a soluble protein at the Centre of Synthetic Biology from Ghent University (UGent), by the iGEM UGent 2016 team. For this reason, InaZ from <em>Pseudomonas syringae</em> is chosen as the INP used in this project. When the INPs are successfully expressed on the bacterial surface of the cells, an inducible promotor is activated to express a protein that causes the cells to leak. This is the so-called bacterial ghost (BG) principle.</p><p>Module engineering also focusses on methods to test the produced agent on its efficiency. To do this, first the ice nucleation activity is tested on supercooled water. The ice nucleation activity efficiency of INPs is generally tested by looking at the lowest sub-zero temperature at which supercooled water starts to freeze. Practically, this is done through a droplet-freezing assay. In this assay ice nuclei are added to a number of water droplets and the freezing temperature of the droplets with ice nuclei is determined by repeating the test with a series of temperatures and looking when the majority of droplets freeze. Since this method is both well described in literature and easy to perform, it is used to test the efficiency of the ice nucleation activity of the vesicles. Other than different temperatures, alternating concentrations of cells are to be tested further to see if this has an effect.</p><p>Finally, a downstream process to recover the bacterial ghosts from the <em>E. Coli</em> culture was developed and optimized. To separate the empty BGs from the cell contents after leaking, centrifugation is used. To be sure there is no DNA left, a treatment with nucleases is used. In a final step, these BGs are to be lyophilized, as this is necessary for the dispersion.</p><h1>The Bacterial Ghost Construct</h1><p>The carriers for the INP in the Vsycle cloud seeding agent are the bacterial ghosts (BGs). BGs are empty cell envelopes of Gram-negative bacteria that expressed the lysis gene E. The functionality of this gene was discovered in 1966, after <em>E. coli</em> cells were infected with the bacteriophage PhiX174 <a href="#citation6">Hutchison et al, 1966</a>. In our project this is done with <em>E. coli</em> cells. After expression protein E makes tunnels in the membrane of the bacteria causing it to expel its cytoplasmatic contents. This kills the bacteria in the process. However, any periplasmatic components, e.g. membrane bound proteins, that are associated with the empty envelope remain intact.</p><p>Unlike many other lytic proteins, protein E has no enzymatic activity <a href="#citation13">Markert et al, 1965</a>;<a href="#citation16">Rodríguez-Rubio et al, 2016</a>. It is a membrane protein that can oligomerize to form the transmembrane tunnels, usually at the centre or the poles of the bacteria <a href="#citation18">Witte et al, 1992</a>. To achieve this, the protein uses a hydrophobic region at its N-terminal end to integrate itself in the inner membrane, leaving the C-terminal end in the cytoplasm. After a conformational change, the C-terminal domain is translocated to the periplasmatic space. Finally, the C-terminal domain is exposed to the cell surface, inducing the fusion of the inner and outer membranes and sealing off the periplasmatic space. This tunnel structure is enforced further by oligomerization. A schematic visualization is shown in Figure 1 <a href="#citation10">Langemann et al, 2010</a>.</p><div class="image"><img src="../assets/img/Engineering_fig1.png" alt="Mechanism of lysis induced by protein E (Langemann et al, 2010)"><p>Figure 1: Mechanism of lysis induced by protein E (Langemann et al, 2010)</p></div><p>Through the tunnel, up to 90% of the cytoplasmatic content is released in the extracellular space <a href="#citation7">Kassmannhuber et al, 2017</a>. The driving force for this is the osmotic pressure difference between the cytoplasm and the extracellular environment.</p><h1>Experiments</h1><h3>Construct design &amp; cloning strategy</h3><div class="image"><img src="../assets/img/Engineering_fig2.png" alt="Schematic representation of the design of the construct"><p>Figure 2: Schematic representation of the design of the construct</p></div><p>In order to grow many bacterial ghosts expressing INPs on their surface, first the genetic construct was designed. A plasmid with both the INP and protein E gene is constructed . The INPs will be expressed on the surface of the bacteria, while protein E will ensure the death of the bacteria. First, a backbone with a pBAD arabinose inducible promoter, ribosome binding site (RBS) and terminator is made. This is done through PCR using <strong>1_BBa_K584000</strong> as template and two primers: <strong>1_K58400_BB_Fw</strong> and <strong>1_K58400_BB_Rv</strong>. This results in the backbone <strong>1_BBa_K584000_Backbone</strong>.</p><p>Next, a CPEC (Circular Polymerase Extension Cloning) is done using <strong>1_BBa_K584000_Backbone</strong> and <strong>2_Protein E + overhangs</strong>, which results in a new plasmid: <em><em>3_pSB13C</em>[PBAD_ProtE]</em>_. In order to achieve this, first <strong>2_Protein E + overhangs</strong> are made using <strong>2_Protein E</strong> as template on which 2 overlapping primers (<strong>2_ProtE_overhang_Fw</strong> and <strong>2_ProtE_overhang_Rv</strong>) are attached. The sticky ends on the protein E gene are homologous with the sticky end on the backbone making annealing possible. CPEC was chosen because it has multiple benefits over BioBricking. CPEC is a method that works without restriction enzymes and instead is based on homologies. On top of that, CPEC does not cause so called scars of left-over sequences the way the BioBrick system does. This means, contrary to the BioBrick system, that the whole reaction can be done in one eppendorf tube.</p><p>Finally, the backbone with the protein E gene (<strong><em>3_pSB13C</em>[PBAD_ProtE]</strong>) can be used in the BioBrick system together with the cut-out of the INP gene (<strong>4_BBa_K1896011</strong>) in order to get the new BioBrick containing both inaZ (INP gene) and protE (Protein E gene) on the same backbone.</p><div class="image"><img src="../assets/img/Engineering_vec1.png" alt="3_pSB13C[PBAD_ProtE]"><p>Figure 3: 3_pSB13C[PBAD_ProtE]</p></div><div class="image"><img src="../assets/img/Engineering_vec2.png" alt="4_BBa_K1896011"><p>Figure 4: 4_BBa_K1896011</p></div><div class="image"><img src="../assets/img/Engineering_vec3.png" alt="1_BBa_K584000"><p>Figure 5: 1_BBa_K584000</p></div><p>The genbank files of all the backbones, plasmids and primers can be downloaded here:</p><a class="btn btn-primary" href="../assets/downloads/Vsycle_genbankfiles.zip" download>Click to download</a><h3>Transformation – Antibiotic selection – Colony PCR – Illumina sequencing</h3><p>After the construct has been made, transformation of the <em>E. Coli</em> cells will be done by electroporation. The purpose of electroporation is to introduce the DNA construct into the cells. When performing the transformation experiments, two controls are taken into account: a negative and a positive control. Non-transformed <em>E. coli</em> cells are used as a negative control while <em>E. coli</em> cells with a non-recombinant vector are used as a positive control. Both the recombinant cells and the positive control are transformed. Subsequently, the transformed cells, the negative and the positive control are plated. Because the negative control are cells that do not contain a vector, they do not have antibiotic resistance and should not grow on LB-agar/Chloroamphenicol medium <a href="#citation12">Maervoet et al., 2019</a>. All possible outcomes regarding growth of the samples and the controls including possible causes are summarized in the Table below (Table 1).</p><table><caption id="table1overviewofpossibleoutcomesregardinggrowthofthesamplesandcontrols">Table 1: Overview of possible outcomes regarding growth of the samples and controls.</caption><thead><tr><th style="text-align:left">Sample</th><th style="text-align:left">Positive control</th><th style="text-align:left">Negative control</th><th style="text-align:left">Causes</th></tr></thead><tbody><tr><td style="text-align:left">Growth</td><td style="text-align:left">Growth</td><td style="text-align:left">Growth</td><td style="text-align:left">- Antibiotic is not active<br/>- One did not work in a sterile way<br/>- Contamination</td></tr><tr><td style="text-align:left">Growth</td><td style="text-align:left">Growth</td><td style="text-align:left">No Growth</td><td style="text-align:left">- Test succeeded<br/>- Antibiotic is active<br/>- No contamination</td></tr><tr><td style="text-align:left">No Growth</td><td style="text-align:left">Growth</td><td style="text-align:left">No Growth</td><td style="text-align:left">- Antibiotic is active<br/>- No contamination<br/>- Cloning failed</td></tr><tr><td style="text-align:left">No Growth</td><td style="text-align:left">No Growth</td><td style="text-align:left">Growth</td><td style="text-align:left">- Antibiotic is not active<br/>- Contamination<br/>- Transformation failed; cells died</td></tr><tr><td style="text-align:left">No Growth</td><td style="text-align:left">No Growth</td><td style="text-align:left">No Growth</td><td style="text-align:left">- Antibiotic is active<br/>- No contamination<br/>- Transformation failed; cells died</td></tr></tbody></table><p>After the transformation has been performed and the plating is done, it must be checked that the plasmid contains both the INP and the lysis gene. The plasmid used for cloning contains an antibiotic marker for chloroamphenicol. By plating the bacteria on LB agar/Chloroamphenicol after cloning, a first selection can be made. Bacteria that contain a vector, either a linear vector or just an insert, will not grow on these media as they do not contain the chloroamphenicol resistance gene or because transcription processes cannot be performed. Transformed <em>E. Coli</em> cells with a circular vector can grow. However, this vector can be a vector without an insert, a vector with the desired or the undesired insert <a href="#citation12">Maervoet et al., 2019</a>. Consequently, it is important to detect this here as well. This can be done by applying colony PCR. This technique determines the length of the insert. In this case, the vector contains two genes: the gene for INP production and the lysis gene. However, only one amplification product will be made that involves both inserts.</p><p>Before colony PCR can be performed, a number of colonies need to be picked up from the LB/Chloroamphenicol plates. It is important to regrow these colonies and to label the petri dishes properly. This way, after the <em>E. coli</em> cells with the desired insert are identified, one can work further with the right colony. This is shown schematically in the Figure below (Figure 3).</p><div class="image"><img src="../assets/img/Engineering_fig3.png" alt="Picking up a colony and labelling plates before starting the colony PCR"><p>Figure 6: Picking up a colony and labelling plates before starting the colony PCR</p></div><p>To perform colony PCR, vector specific primers are used. These are primers that are complementary to the vector and flank its multiple cloning site (MCS). As mentioned before, colony PCR can be used to differentiate between a vector without insert or a vector with (un)desired insert. This is because the amplification product after applying the technique differs in length. The amplification product of a vector with (un)desired insert will always be longer than the amplification product of a non-recombinant vector. By separating the obtained fragments by gel electrophoresis, it can be determined whether or not a cell contains an insert <a href="#citation9">Kyndt, Van Damme, &amp; De Mey, 2019</a>; <a href="#citation12">Maervoet et al., 2019</a>. For this purpose, the length of the genes and the amplified part of the MCS must be determined in advance and a comparison must be made with a molecular ladder. It is known that the INP gene is 3603 bp long and the protein E gene of Phage phiX174 is 279 bp long <a href="#citation2">Brempt &amp; Van Hove, 2016</a>; <a href="#citation17">Wang, 2013</a>. If a band is found on the gel that deviates from the size of the desired insert, no insert or another insert is cloned into the plasmid. If a band matches the desired size, it can be considered that these <em>E. coli</em> cells contain the desired insert. Then this colony will be picked up again and grown up on a new LB agar/Chloroamphenicol plate to make sure the right colony can be used later on in the fermentation process <a href="#citation12">Maervoet et al., 2019</a>.</p><div class="image"><img src="../assets/img/Engineering_fig4.png" alt="Picking up and regrowing the right colony after performing the colony PCR."><p>Figure 7: Picking up and regrowing the right colony after performing the colony PCR.</p></div><p>Even if the cloned genes turn out to be the desired insert based on its size, this does not guarantee the absence of any mutations. Consequently, this also needs to be investigated and the plasmid is isolated from the appropriate <em>E. coli</em> cells. The obtained insert is then sequenced by the external company Macrogen Europe.</p><h3>Fermentation</h3><p>The bacteria are grown through fermentation. This is done based on the procedure described by Langemann et al. <a href="#citation11">Langemann et al., 2010</a>. First, inoculation is performed, followed by a growth phase. Here the bacteria are able to grow in the appropriate conditions for 90 minutes. As a growth medium, a lysogeny broth (LB) medium is chosen. This is a complex medium, that is often used and results in a lot of biomass. However, M9 is cheaper than LB medium so that this can be used as an alternative to grow cells. Parameters that have to be controlled during the growth phase are listed in the table below (Table 2).</p><table><caption id="table2overviewofparametersthathavetobecontrolledduringthefermentationndnotdefined">Table 2: Overview of parameters that have to be controlled during the fermentation. N.D. = Not defined</caption><thead><tr><th style="text-align:left">Parameter</th><th style="text-align:left">Value</th><th style="text-align:left">Explanation</th></tr></thead><tbody><tr><td style="text-align:left">Temperature</td><td style="text-align:left">37°C</td><td style="text-align:left">Lower temperatures will result in more expression, since the folding is better and less inclusion bodies are formed. Nonetheless, this is not optimal for our setup and a temperature of 37°C is selected.</td></tr><tr><td style="text-align:left">pH</td><td style="text-align:left">6.9</td><td style="text-align:left">Optimal pH for this fermentation.</td></tr><tr><td style="text-align:left">Dissolved oxygen (saturation level)</td><td style="text-align:left">20%</td><td style="text-align:left">20% saturation is used to create aerobic conditions in order to avoid anaerobic fermentation.</td></tr><tr><td style="text-align:left">Stirring</td><td style="text-align:left">N.D.</td><td style="text-align:left">Stirring level has to be adjusted according to the growth phase.</td></tr><tr><td style="text-align:left">Aeration</td><td style="text-align:left">N.D.</td><td style="text-align:left">Aeration will be adjusted according to the growth phase.</td></tr></tbody></table><p>Secondly, a constitutive promoter is used. This means that the INP expression starts immediately, without the need of an inducer. Finally, cell lysis is induced when the stationary phase of growth started. This is determined using optical density (OD) coupled with cell dry weight (CDW) or colony-forming unit (CFU). Examples of these relationships for an <em>E. Coli</em> strain are given below.</p><p>$$CDW [g/l] = 0.386 * OD [arb. unit] - 0.0013$$</p><p>$$CFU [\text{#}/mL] = 2.72 * 10^{17} * OD [arb. unit] - 2.7*10^{16}$$</p><p>Important to remember is that these equations have to be determined again for our specific strain of <em>E. coli</em>. After cell lysis is initiated, a drop in OD value will be detected. This is due to the fact that bacteria get more translucent after lysis. For our setup, OD values will be determined at 600 nm <a href="#citation10">Langemann et al., 2010</a>.</p><p>During the growth phase, a sample is taken every hour to follow up all the batches. For these samples, the OD and the CFU/CDW are determined. This is done to follow up the growth rate. In practice, this means a dilution series is set up form 10-1 until 10-6. The OD is then determined for this dilution series and the dilutions 10-3 up until 10-6 get plated. As such, the OD can be related to another type of metric like CFU or CDW in order to get a quantitative estimation of the growth <a href="#citation10">Langemann et al., 2010</a>.</p><p>During the lysis phase of the fermentation additional samples of the culture are taken. Of these samples three things are measured: CFU, OD<sub>600</sub> and transparency under a light microscope. This to follow the progress of the lysis phase. Both determining the OD<sub>600</sub> of the culture and studying the bacteria under a light microscope are methods commonly used as a qualitative indicator of the E-mediated lysis quality <a href="#citation10">Langemann et al, 2010</a>. These methods rely on the fact that the bacteria become translucent after they have been lysated, which can be seen under a light microscope. Successful E-lysis is characterised by a drop of the OD<sub>600</sub> over the course of the fermentation.</p><p>After fermentation, the batch of cells must undergo further downstream processing. First of all, the biomass of the medium is separated by means of Tangential Flow Filtration (TFF) <a href="#citation10">Langemann et al., 2010</a>. As an alternative to TFF, centrifugation (at 10 000 g) can also be used on a smaller scale <a href="#citation1">Amara, Salem-Bekhit, &amp; Alanazi, 2013</a>; <a href="#citation15">Rabea et al., 2018</a>. In order to guarantee that the product does not contain GMO’s, all remaining viable cells and DNA have to be destroyed. This can be done by using β-propiolactone <a href="#citation1">Amara et al., 2013</a>; [Hajam, Dar, Won, &amp; Lee, 2017])(#citation4). This chemical is a water-soluble organic component with alkylating properties <a href="#citation11">Langemann, 2011</a>. After reaction with the nucleic acids of the DNA, the structure of purine and pyrimidine residues will be altered resulting in nicks or causing the formation of cross-links between the two DNA helices <a href="#citation14">Perrin &amp; Morgeaux, 1995</a>. Instead of β-priopolactone, hydrogen peroxide (H2O2) can also be used to inactivate remaining DNA <a href="#citation4">Hajam et al., 2017</a>. Subsequently, the biomass is separated from other cell material and chemicals by applying diafiltration. Finally, the cells are washed several times with sterile water <a href="#citation10">Langemann et al., 2010</a>.</p><h3>E-lysis efficiency</h3><p>Another thing that needs to be tested, is whether the <em>E. coli</em> cells have been successfully transformed to bacterial ghosts and how efficiently this happened. To do this, a sample of the bacterial culture before induction of the BG gene is taken. After performing a dilution series, 100 µl of each dilution is plated immediately on LB medium. The same is done after induction. In both cases the CFU is determined to calculate the lysis ratio (LE) with the following formula:</p><p>$$LE=(1-\frac{CFU_{t}}{CFU_{t_{0}}}*100\text{%})$$</p><p>As a negative control, the same experiment is done with <em>E. coli</em> cells which are not lysed.</p><p>The different outcomes for this experiment and their interpretations are described in Table 3.</p><table><caption id="table3thedifferentoutcomesforthisexperimenttheirinterpretationsandpossibleinvestigationsinthecaseofunwantedresults">Table 3: The different outcomes for this experiment, their interpretations and possible investigations in the case of unwanted results</caption><thead><tr><th style="text-align:left">Efficiency</th><th style="text-align:left">Interpretation</th><th style="text-align:left">Further investigation</th></tr></thead><tbody><tr><td style="text-align:left">&gt; 98-99%</td><td style="text-align:left">- Experiment has succeeded</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">&lt; 98-99%</td><td style="text-align:left">- Fermentation after induction was not long enough<br/>- Aberrant pH<br/>- Aberrant fermentation temperature<br/>- Aberrant oxygen level<br/>- Aberrant pressure<br/>- Aberrant amount of nutrients</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">+/- 0%</td><td style="text-align:left">- Lysis has not occurred, protein E might not have been formed</td><td style="text-align:left">- Studying on RNA level (QPCR)</td></tr></tbody></table><p>During the lysis phase of the fermentation additional samples of the culture are taken. Of these samples three things are measured: CFU, OD<sub>600</sub> and transparency under a light microscope. This is to follow the progress of the lysis phase. Both determining the OD<sub>600</sub> of the culture and studying the bacteria under a light microscope are methods commonly used as a qualitative indicator of the E-mediated lysis quality <a href="#citation10">Langemann et al, 2010</a>. These methods rely on the fact that the bacteria become translucent after they have been lysated, which can also be seen under a light microscope. Successful E-lysis is characterised by a drop of the OD<sub>600</sub> over the course of the fermentation.</p><h3>Testing the expression of the INP</h3><p>To test if the INPs are successfully expressed on the outer membrane of the <em>E. coli</em> cells (A), tests using supercooled water can be performed. More specifically, this is done by transferring 50 µL of the fermentation media in a test tube containing 50 µL of supercooled water at -6°C if an ethanol cooling bath is used. If the supercooled water freezes after adding the bacteria, this means that the INPs have been expressed. To prove not just any protein can initiate ice nucleation, a negative control is used. For this purpose, <em>E. coli</em> cells that do not express INP but GFP (B) are tested. These should not result in the freezing of the supercooled water. The different possible outcomes for this experiment and their interpretations are described in Table 4.</p><table><caption id="table4thedifferentoutcomesforthisexperimenttheirinterpretationsandpossibleinvestigationsinthecaseofunwantedresults">Table 4: The different outcomes for this experiment, their interpretations and possible investigations in the case of unwanted results</caption><thead><tr><th style="text-align:left">Result</th><th style="text-align:left">Interpretation</th><th style="text-align:left">Additional tests</th></tr></thead><tbody><tr><td style="text-align:left">Supercooled water freezes after adding A, but not after adding B</td><td style="text-align:left">Experiment has succeeded, INP is expressed on the membrane</td><td style="text-align:left"></td></tr><tr><td style="text-align:left">Supercooled water freezes after adding both A and B</td><td style="text-align:left">- Supercooled water contains impurities that initiate ice nucleation<br/>- Supercooled water has been moved to abruptly, causing ice nucleation</td><td style="text-align:left">Filtration of the supercooled water</td></tr><tr><td style="text-align:left">Supercooled water does not freeze after adding both A and B</td><td style="text-align:left">INP has not been expressed:<br/>- Problem with promotor<br/>- Problem with induction of promotor<br/>- Problem with protein folding<br/>- Protein is unable to reach the outer membrane<br/>- Protein could not be formed due to lack of nutrients</td><td style="text-align:left">Studying the protein expression:<br/>- Proteomics<br/>- SDS page<br/>- Adding a His-tag to the INP and using compatible antibodies<br/><br/>Studying on RNA level<br/>- QPCR<br/><br/>Studying under microscope to see if inclusion bodies have been formed</td></tr><tr><td style="text-align:left">Supercooled water freezes after adding B, but not after adding A</td><td style="text-align:left">Human error</td><td style="text-align:left"></td></tr></tbody></table><h3>Ice nucleation efficiency</h3><p>To test the ice nucleation efficiency of the <em>E. coli</em> cells/BGs with INPs on their membrane and the concentrations needed to freeze supercooled water, a droplet freezing assay is used. This method is used in general to determine the ice nucleation efficiencies of ice nuclei. The assay used here is based on the droplet freezing assay used by <a href="#citation7">Kassmannhuber et al, 2017</a>. In this assay, a series of tenfold dilutions of the BGs (ranging from 5 x 10<sup>8</sup> cells ml<sup>-1</sup> to 5 x 10<sup>4</sup> cells ml<sup>-1</sup>) is made. From each of these dilutions, 54 droplets (10 µl) are placed on a sterile aluminium plate with a hydrophobic film. After using a cooling unit to adjust the temperature of the plate the droplets are supercooled. By looking at the number of droplets that are frozen at a certain temperature, the ice nucleation efficiency can be determined. For this a temperature range between -13 an -2 °C is used, raised at a constant interval of 0.5 or 1 °C. Each time, the temperature on the plate is maintained for 30 seconds.</p><p>Two parameters that characterise the ice nucleation efficiency can be determined from this experiment. The first one is the median freezing temperature (T<sub>50</sub>). This is the temperature where 50% of the droplets are frozen. This can be calculated from the following formula <a href="#citation8">Kishimoto et al., 2014</a>:</p><p>$$T_{50}=\frac{T_1+(T_2-T_1)(2^{-1}n-F_1)}{F_2-F_1}$$</p><p>Where F1 and F2 are the number of frozen drops at temperature T1 and adjacent temperature T2, and are just below and above 50% of the total number of tested drops (n).</p><p>The second parameter is the cumulative number N(T), which describes the number of ice nuclei ml<sup>-1</sup> at a given temperature. It is calculated from the following formula <a href="#citation3">Govindarajan and Lindow, 1988</a>:</p><p>$$N(T)=-\ln{f}*\frac{10^D}{V}$$</p><p>f is the fraction of droplets not frozen at a specific temperature T. f is described as total number of droplets used divided by unfrozen droplets (N<sub>0</sub>/ N<sub>U</sub>). V refers to the volume of each droplet used (10 µl). D is the number of 10^<sup>-1</sup> serial dilutions of the original suspension. To obtain the nucleation frequency (NF) per cell by dividing ice nuclei/ml through cell density (cell/ ml), N(T) is normalized for the number of cells present in each solution.</p></article><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Amara, A. A., Salem-Bekhit, M. M., &amp; Alanazi, F. K. (2013).</p><cite>Sponge-like: A new protocol for preparing bacterial ghosts.</cite><p><span class="journalTitle">The Scientific World Journal. </span><span class="journalInfo"></span></p><a class="in-text" href="https://doi.org/10.1155/2013/545741" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Sponge-like: A new protocol for preparing bacterial ghosts." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Brempt, M., &amp; Van Hove, B. (2016).</p><cite>InaZ (GG safe).</cite><p><span class="details">Retrieved on August 02, 2020. from </span><a class="in-text" href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1896002" target="_blank" rel="noopener">http://parts.igem.org/wiki/index.php?title=Part:BBa_K1896002</a></p><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Govindarajan, A. G., &amp; Lindow, S. E. (1988).</p><cite>Size of bacterial ice-nucleation sites measured in situ by radiation inactivation analysis.</cite><p><span class="journalTitle">Proceedings of the National Academy of Sciences </span><span class="journalInfo">85(5), 1334-1338.</span></p><a class="in-text" href="https://doi.org/10.1080/21655979.2017.1284712" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Size of bacterial ice-nucleation sites measured in situ by radiation inactivation analysis." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Hajam, I. A., Dar, P. A., Won, G., &amp; Lee, J. H. (2017).</p><cite>Bacterial ghosts as adjuvants: Mechanisms and potential.</cite><p><span class="journalTitle">Veterinary Research. </span><span class="journalInfo"></span></p><a class="in-text" href="https://doi.org/10.1186/s13567-017-0442-5" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Bacterial ghosts as adjuvants: Mechanisms and potential." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Han, Y. J., Song, H., Lee, C. W., Ly, N. H., Joo, S.-W., Lee, J. H., Kim, S.-J., and Park, S. (2017).</p><cite>Biophysical characterization of soluble pseudomonas syringae ice nucleation protein inaz fragments.</cite><p><span class="journalTitle">International journal of biological macromolecules </span><span class="journalInfo">94:634–641.</span></p><a class="in-text" href="https://doi.org/10.1016/j.ijbiomac.2016.10.062" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Biophysical characterization of soluble pseudomonas syringae ice nucleation protein inaz fragments." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Hutchison III, C. A., &amp; Sinsheimer, R. L. (1966).</p><cite>The process of infection with bacteriophage ΦX174: X. Mutations in a ΦX lysis gene.</cite><p><span class="journalTitle">Journal of molecular biology </span><span class="journalInfo">18(3), 429-IN2</span></p><a class="in-text" href="https://doi.org/10.1016/0042-6822(65)90256-4" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= The process of infection with bacteriophage ΦX174: X. Mutations in a ΦX lysis gene." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">Kassmannhuber, J., Rauscher, M., Schöner, L., Witte, A., &amp; Lubitz, W. (2017).</p><cite>Functional display of ice nucleation protein InaZ on the surface of bacterial ghosts.</cite><p><span class="journalTitle">Bioengineered </span><span class="journalInfo">8(5), 488-500.</span></p><a class="in-text" href="https://doi.org/10.1080/21655979.2017.1284712" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Functional display of ice nucleation protein InaZ on the surface of bacterial ghosts." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li><li id="citation8"><p class="author">Kishimoto, T., Yamazaki, H., Saruwatari, A., Murakawa, H., Sekozawa, Y., Kuchitsu, K., ... &amp; Ishikawa, M. (2014).</p><cite>High ice nucleation activity located in blueberry stem bark is linked to primary freeze initiation and adaptive freezing behaviour of the bark.</cite><p><span class="journalTitle">AoB Plants </span><span class="journalInfo">6.</span></p><a class="in-text" href="https://doi.org/10.1093/aobpla/plu044" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= High ice nucleation activity located in blueberry stem bark is linked to primary freeze initiation and adaptive freezing behaviour of the bark." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext8">Back to text</a></li><li id="citation9"><p class="author">Kyndt, T., Van Damme, E., &amp; De Mey, M. 2019</p><cite>Gene technology</cite><p><span class="details"></span></p><a class="in-text" href="#intext9">Back to text</a></li><li id="citation10"><p class="author">Langemann, T., Koller, V. J., Muhammad, A., Kudela, P., Mayr, U. B., &amp; Lubitz, W. (2010).</p><cite>The bacterial ghost platform system.</cite><p><span class="journalTitle">Bioengineered Bugs. </span><span class="journalInfo"></span></p><a class="in-text" href="https://doi.org/10.4161/bbug.1.5.12540" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The bacterial ghost platform system." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext10">Back to text</a></li><li id="citation11"><p class="author">Langemann, T. (2011).</p><cite>Process development for industrial scale bacterial ghost production.</cite><p><span class="journalTitle">Bioengineered. </span><span class="journalInfo"></span></p><a class="in-text" href="http://othes.univie.ac.at/15287/1/2011-05-24_0847940.pdf#page=81" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Process development for industrial scale bacterial ghost production." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext11">Back to text</a></li><li id="citation12"><p class="author">Maervoet, V., João Maurício da Fonseca, M., &amp; Watson, H. 2019</p><cite>Gene technolgy practical excercises.</cite><p><span class="details"></span></p><a class="in-text" href="#intext12">Back to text</a></li><li id="citation13"><p class="author">Markert, A., &amp; Zillig, W. (1965).</p><cite>Studies on the lysis of Escherichia coli C by bacteriophage φX174.</cite><p><span class="journalTitle">Virology </span><span class="journalInfo">25(1), 88-97</span></p><a class="in-text" href="https://doi.org/10.1016/0042-6822(65)90256-4" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Studies on the lysis of Escherichia coli C by bacteriophage φX174." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext13">Back to text</a></li><li id="citation14"><p class="author">Perrin, P., &amp; Morgeaux, S. (1995).</p><cite>Inactivation of DNA by β-propiolactone.</cite><p><span class="journalTitle">Biologicals.</span> <span class="journalInfo"></span></p><a class="in-text" href="https://doi.org/10.1006/biol.1995.0034" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Inactivation of DNA by β-propiolactone." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext14">Back to text</a></li><li id="citation15"><p class="author">Rabea, S., Salem-Bekhit, M. M., Alanazi, F. K., Yassin, A. S., Moneib, N. A., &amp; Hashem, A. E. M. (2018).</p><cite>A novel protocol for bacterial ghosts’ preparation using tween 80.</cite><p><span class="journalTitle">Saudi Pharmaceutical Journal. </span><span class="journalInfo"></span></p><a class="in-text" href="https://doi.org/10.1016/j.jsps.2017.12.006" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= A novel protocol for bacterial ghosts’ preparation using tween 80." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext15">Back to text</a></li><li id="citation16"><p class="author">Rodríguez-Rubio, L., Gutiérrez, D., Donovan, D. M., Martínez, B., Rodríguez, A., &amp; García, P. (2016).</p><cite>Phage lytic proteins: biotechnological applications beyond clinical antimicrobials.</cite><p><span class="journalTitle">Critical reviews in biotechnology </span><span class="journalInfo">36(3), 542-552.</span></p><a class="in-text" href="https://doi.org/10.3109/07388551.2014.993587" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q= Phage lytic proteins: biotechnological applications beyond clinical antimicrobials." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext16">Back to text</a></li><li id="citation17"><p class="author">Wang, J. (2013).</p><cite>Protein E of Phage phiX174.</cite><p><span class="details">Retrieved on August 29, 2020. from </span><a class="in-text" href="http://parts.igem.org/Part:BBa_K1054000" target="_blank" rel="noopener">http://parts.igem.org/Part:BBa_K1054000</a></p><a class="in-text" href="#intext17">Back to text</a></li><li id="citation18"><p class="author">Witte, A., Wanner, G., Sulzner, M., &amp; Lubitz, W. (1992).</p><cite>Dynamics of PhiX174 protein E-mediated lysis of Escherichia coli.</cite><p><span class="journalTitle">Archives of microbiology </span><span class="journalInfo">157(4), 381-388.</span></p><a class="in-text" href="https://doi.org/10.1007/BF00248685" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Dynamics of PhiX174 protein E-mediated lysis of Escherichia coli." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext18">Back to text</a></li></ol></article></div></div></div></main><footer><div class="clearfix d-flex justify-content-between"><div class="social d-flex flex-column"><div class="social-item"><a class="facebook" href="https://www.facebook.com/iGEMUGent2020" target="_blank"></a></div><div class="social-item"><a class="twitter" href="https://twitter.com/igem_ghent" target="_blank"></a></div><div class="social-item"><a class="instagram" href="https://www.instagram.com/igemugent2020" target="_blank"></a></div></div><div class="sponsor d-flex flex-column"><div class="sponsor-item lg"><a href="https://reviverestore.org" target="_blank"><img src="../assets/img/sponsor_rnr.png"></a></div><div class="sponsor-item"><a href="http://csb.mystrikingly.com" target="_blank"><img src="../assets/img/sponsor_cfsb.png"></a></div></div><div class="sponsor d-flex flex-column"><div class="sponsor-item"><a href="https://www.dewatergroep.be" target="_blank"><img src="../assets/img/sponsor_watergroep.png"></a></div></div><div class="sponsor d-flex flex-column"><div class="sponsor-item lg"><a href="https://www.ugent.be" target="_blank"><img src="../assets/img/sponsor_ugent.png"></a></div></div><div class="sponsor d-flex flex-column"><div class="sponsor-item lg"><a href="https://www.ugent.be/bw/" target="_blank"><img src="../assets/img/sponsor_faculteit.png"></a></div><div class="sponsor-item"><a href="http://www.biobix.be" target="_blank"><img src="../assets/img/sponsor_biobix.png"></a></div></div></div></footer><script src="../content.bundle.js"></script></body></html>